objgraph.py

```diff
@@ -564,8 +566,8 @@ def find_chain(obj, predicate, edge_func, max_depth=20, extra_ignore=()):
     ignore.add(id(depth))
     ignore.add(id(parent))
     ignore.add(id(ignore))
-    ignore.add(id(sys._getframe()))  # this function
-    ignore.add(id(sys._getframe(1))) # find_chain/find_backref_chain, most likely
+    ignore.add(id(sys._getframe()))   # this function
+    ignore.add(id(sys._getframe(1)))  # find_chain/find_backref_chain
```

Comment:
Why did you remove "most likely" here but not from the comment below?

(The reason for "most likely" is that there was nothing preventing the user from invoking find_chain() directly, although that was unlikely because why would one do that?  Maybe it's now even less likely when that function is explicitly marked private, in which case I'm fine with removing the "most likely" comment here, but then it should also be removed from the comment in `_show_graph`.)